### DISCUSSION TITLE
(non self-hosted) Is there a way to not have articles older than 30 days removed?

### DISCUSSION THREAD

**Discussion by AaronKnowls** [Athou/commafeed#1361] [Q&A]
**Is your feature request related to a problem? Please describe.**
I'd like to not have older than 30 days articles automatically removed.

**Describe the solution you'd like**
To not remove any articles until I have marked them as read.

**Describe alternatives you've considered**
- I can't find a way to circumvent this, because of the way RSS is useful to me.
- Other services that have this feature are far worst in everything else compared to commafeed.

**Additional context**
I thoroughly searched through the issues and didn't find anything related. I tried my best. I hope I'm not annoying anyone here by posting this, I just can't find a way and this is the only feature keeping me from using this amazing piece of code.

**Comment by Athou** [comment]
Unfortunately, this is required for commafeed.com to remain online. Keeping statuses for longer would be too costly. The only solution would be to host a commafeed instance yourself and configure it to keep statuses for a longer period of time.

**Comment by AaronKnowls** [comment]
Thank you so much for taking the time to answer. And thank you for your awesome work.
